File: code-coaches/easy/duct-tape/DuctTape.cs
using System;

namespace DuctTape
{
    class Program
    {
        static int GetArea(int height, int width)
        {
            return height * width;
        }

        static int ToInches(int feet)
        {
            return feet * 12;
        }

        static int GetDuctTapesNeeded(int height, int width)
        {
            // Everything is calculated in inches
            int ductTapeHeight = 720;
            int ductTapeWidth = 2;

            int doorAreaBothSides = GetArea(height, width) * 2;
            int ductTapeArea = GetArea(ductTapeHeight, ductTapeWidth);

            int ductTapesNeeded = (int)Math.Ceiling((double)doorAreaBothSides / ductTapeArea);
            return ductTapesNeeded;
        }

        static void Main(string[] args)
        {
            int doorHeightFeet = int.Parse(Console.ReadLine());
            int doorWidthFeet = int.Parse(Console.ReadLine());

            int doorHeightInches = ToInches(doorHeightFeet);
            int doorWidthInches = ToInches(doorWidthFeet);

            Console.WriteLine(GetDuctTapesNeeded(doorHeightInches, doorWidthInches));
        }
    }
}

File: code-coaches/easy/duct-tape/DuctTape.swift
import Foundation

func getArea(height: Int, width: Int) -> Int {
    return height * width
}

func toInches(feet: Int) -> Int {
    return feet * 12
}

func getDuctTapesNeeded(height: Int, width: Int) -> Int {
    // Everything is calculated in inches
    let ductTapeHeight = 720
    let ductTapeWidth = 2

    let doorAreaBothSides = getArea(height: height, width: width) * 2
    let ductTapeArea = getArea(height: ductTapeHeight, width: ductTapeWidth)

    let ductTapesNeeded = Int(ceil(Double(doorAreaBothSides) / Double(ductTapeArea)))
    return ductTapesNeeded
}

func main() {
    let doorHeightFeet = Int(readLine()!)!
    let doorWidthFeet = Int(readLine()!)!
        
	let doorHeightInches = toInches(feet: doorHeightFeet)
	let doorWidthInches = toInches(feet: doorWidthFeet)

	print(getDuctTapesNeeded(height: doorHeightInches, width: doorWidthInches))
}

main()
